### ISSUE TITLE
Include sounds

### ISSUE THREAD

**Issue by vincenicoara** [UCSB-CS56-Projects/cs56-games-battleship#26]
- Include different sounds for all of the below: 
  - When you drop your ship
  - When you try to drop your ship in an invalid area
  - When you HIT your opponents ship
  - When you DONT HIT your opponents ship
  - When your opponent HITS your ship
  - When your opponent DOESNT HIT your ship
  - When you win the game
  - When you lose the game
- Also include a checkbox button on the game screen that enables you to turn the sounds on and off during the game.

**550 points** (previous value was 450)

**Comment by joesong96** [comment]
All of these were completed except for when the opponent hits/doesn't hit the ship because the sound plays and overlaps with your shooting sound when you're playing against a computer (because the computer makes its turn almost instantly).

For some reason having to do with the audio file formats, the sound doesn't work on CSIL and only works if you run it on your own machine.

I'd like to request to have this worth more points if possible. It came out to be a lot of work and took an entire day because I needed to learn how to have ant include sound files when creating a jar, how to get the sound files to play, and I also needed to create my own sound files and find the others online, and then change their bitrates so that they would play better.
